README.md

```diff
@@ -163,3 +163,65 @@ Interested in Integrating SocialConnect, get in touch by filling this [form](htt
 ## 📣 Feedback
 
 **SocialConnect is in beta**! Help us improve by sharing feedback on your experience in the Github Discussion section. You can also open an issue or a PR directly on this repo.
+
+## FAQ
+
+<details>
+  <summary>What is an Identifier?</summary>
```

Comment:
Can we split this up into the specific terms used in the SDK: `plaintextIdentifier`, `obfuscatedIdentifier` and making the differences clear? (Maybe even some text examples of what each of these looks like)

[This glossary](https://www.notion.so/clabsco/ASv2-User-Flows-f508062cc99d4376af4e161859f2e745?pvs=4#8fbbf28172794aaf843c92e97078b7fe) may be helpful for reusing some definitions here + elsewhere!

Reply:
Will create a Mermaid diagram instead to explain the terminologies.